diff -ur xsldbg-2.1.8-/src/libxsldbg/Makefile.am xsldbg-2.1.8/src/libxsldbg/Makefile.am
--- xsldbg-2.1.8-/src/libxsldbg/Makefile.am	Sat Apr 20 16:46:32 2002
+++ xsldbg-2.1.8/src/libxsldbg/Makefile.am	Sat Apr 20 16:46:59 2002
@@ -1,10 +1,10 @@
 # set the include path found by configure
 # cheet a bit because I need DOCS_PATH defined
 if USE_XSLDBG_AS_THREAD
-INCLUDES= $(all_includes) -DDOCS_PATH=\"$(HTML_DIR)/xsldbg/\" \
+INCLUDES= $(all_includes) -DDOCS_PATH=\"$(HTML_DIR)/\" \
  -DVERSION=\"@VERSION@\" -I .. $(LIBXML_CFLAGS) $(LIBXSLT_CFLAGS) -I $(QTDIR)/include
 else
-INCLUDES= $(all_includes) -DDOCS_PATH=\"$(HTML_DIR)/xsldbg/\" \
+INCLUDES= $(all_includes) -DDOCS_PATH=\"$(HTML_DIR)/\" \
  -DVERSION=\"@VERSION@\" -I .. $(LIBXML_CFLAGS) $(LIBXSLT_CFLAGS)
 endif
 
