diff -ur xsldbg-2.1.8/configure.in xsldbg-2.1.8-/configure.in
--- xsldbg-2.1.8/configure.in	Sun Apr  7 16:08:52 2002
+++ xsldbg-2.1.8-/configure.in	Sat Apr 20 15:40:25 2002
@@ -561,7 +561,9 @@
 	XSLDBG_STATIC_FLAGS=
 	AC_MSG_RESULT(Disabling static build of xsldbg)
 else
-	XSLDBG_STATIC_FLAGS=" -static"
+#	XSLDBG_STATIC_FLAGS=" -static"
+#	we don't want static
+	XSLDBG_STATIC_FLAGS=
 fi
 
 AC_SUBST(XSLDBG_STATIC_FLAGS)
